index.js

```diff
@@ -35,6 +36,8 @@ const regex = /@(jsx|jsxFrag|jsxImportSource|jsxRuntime)\s+(\S+)/g
  * @property {string} [importSource='react']
  * @property {string} [pragma='React.createElement']
  * @property {string} [pragmaFrag='React.Fragment']
+ * @property {boolean} [development=false]
+ * @property {string} [filePath]
```

Comment:
Since this is inserted as the `fileName` property, maybe we should name this option `fileName` as well?